src/components/EventDetails/EventDetails.tsx

```diff
@@ -46,18 +46,18 @@ const EventDetails = ({ eventId }: EventDetailsProps) => {
             <ul className="EventDetails__attendees">
               {eventData?.attendees?.map(attendee => (
                 <li
-                  key={attendee._id.toString()}
+                  key={attendee?._id.toString()}
```

Comment:
There's a similar issue here. In `getEvent.ts:33` we call `db.get(attendee.userId)`. We'd expect every attendee to be a valid user with an ID, but the type indicates it could be `null`. And so we can't confidently refer to the `attendee` here.

Reply:
how about fixing this in the query function, in this case by filtering out nulls
```
  // Get details for each of the attendees.
  const attendeeUsers = (await Promise.all(
    attendees.map(async attendee => await db.get(attendee.userId))
  )).filter(v => v);
```

Reply:
Good idea! I'll get these nulls filtered out so we don't have to worry about rendering what's essentially an error case.